app/models/instrument_processes_instrument.rb

```diff
@@ -4,9 +4,6 @@ class InstrumentProcessesInstrument < ApplicationRecord
   belongs_to :instrument
   belongs_to :instrument_process
 
-  validates :instrument, presence: true
-  validates :instrument_process, presence: true
-
```

Comment:
From Ben - this was a rubocop change -

> Think its because you dont need to validate presence when you have belongs_to (seen above deleted lines). Was a suggested rubocop change